docker/shared_steps/install_php_extensions.sh

```diff
@@ -10,6 +10,17 @@ php_install_gd() {
   eatmydata docker-php-ext-install -j$COMPILE_JOBS $ext
 }
 
+php_install_ldap() {
+  local php=$(which php)
```

Comment:
You can use `$PHP` here, so you don't need to determine the right binary by yourself